ProcessLib/BoundaryCondition/Python/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+add_library(ProcessLibBoundaryConditionPython
+    PythonBoundaryCondition.cpp
+    PythonBoundaryCondition.h
+    PythonBoundaryConditionLocalAssembler.h
+    PythonBoundaryConditionPythonSideInterface.h)
+
+target_compile_definitions(ProcessLibBoundaryConditionPython
+    PUBLIC OGS_USE_PYTHON)
+
+target_link_libraries(ProcessLibBoundaryConditionPython
+    PUBLIC BaseLib MathLib MeshLib NumLib logog)
+
+target_link_libraries(ProcessLibBoundaryConditionPython
+    PRIVATE pybind11::pybind11)
```

Comment:
You can merge this command with the previous one:

```cmake
target_link_libraries(ProcessLibBoundaryConditionPython
    PUBLIC BaseLib MathLib MeshLib NumLib logog
    PRIVATE pybind11::pybind11)
```

Reply:
All pvsm files will be deleted.
Thanks for the hint. I'll change it.